packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -221,7 +221,7 @@ class VerboseLogger extends Logger {
     stopwatch.reset();
 
     String prefix;
-    const int prefixWidth = 12;
+    const int prefixWidth = 8;
```

Comment:
This was intentionally 12. It turns out 8 is not enough for the length of timestamps that occur normally in regular use.

Four characters are lost to boilerplate (+, a space, and ms). We need to be able to show a few hours in milliseconds without the prefix getting all warped. Three hours takes 10,800,000ms which is 8 digits. 8+4=12.

I admit I should have left a comment there. :-)